bindata/network/openshift-sdn/monitor.yaml

```diff
@@ -0,0 +1,74 @@
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  labels:
+    app: sdn
+  name: monitor-sdn
+  namespace: openshift-sdn
+spec:
+  endpoints:
+  - bearerTokenFile: /var/run/secrets/kubernetes.io/serviceaccount/token
+    honorLabels: true
```

Comment:
Do you actually need this? Typically this is only required if you have things like the `namespace`, `pod`, `service`, `job` or `instance` label in the metrics you're exposing. And if you do, we should probably talk about that as well :slightly_smiling_face: .